### DISCUSSION TITLE
Close paren in flipped layout

### DISCUSSION THREAD

**Discussion by Lenbok** [manna-harbour/miryoku#95] [Q&A] [answered]
I had started to learn miryoku with qwerty + vim mode, but decided that flipped was much closer to what I'd been using previously. I'm using a sweep, so getting the third (combo) thumbs aren't as comfortable as the other thumb keys.

With the non-flipped sym layer it's easy to get both open and close paren "()" without having to use the combo thumb, however, when using flipped sym, the close paren is on the combo thumb. I'm wondering if there's an easy way to have the primary thumb key on sym layer ")", just like it is on the non-flipped sym? (Alternatively, the answer could just be to just suck it up and learn to use either use shift-num-0 or sym-thumb3).

**Comment by manna-harbour** [comment, marked as answer]
You can just use shift on Num as you mentioned, and you can hold mods and layer change simultaneously. Sym is more of a convenience to substitute a mod hold for a thumb movement. And the extra open paren on the Sym thumb is just a convenience to match with the close paren.

For flip, parts of the layout are mirrored and some are translated, depending on whether those keys are based on finger or left-right relationships, but there's an argument for both. In the case of the extra open paren on Sym and moving close paren to the tertiary thumb on flip, it's a trade off between keeping the relative positions, and losing the shift mapping with 0. Unfortunately with thumb combos it also means requiring the combo. So keeping close paren on primary and open on tertiary is better in some ways.

There's no built in option to switch, but there's now support for layer customisation, so you could substitute a modified Sym layer. See https://github.com/manna-harbour/miryoku/discussions/85.

Also if you use parens heavily consider adding combos on Base. See https://github.com/manna-harbour/miryoku/issues/94.
